enterprise/coderd/workspaceproxy.go

```diff
@@ -325,7 +412,20 @@ func (api *API) workspaceProxyRegister(rw http.ResponseWriter, r *http.Request)
 	var (
 		ctx   = r.Context()
 		proxy = httpmw.WorkspaceProxy(r)
+		// TODO: This audit log does not work because it has no user id
+		// associated with it. The audit log commitAudit() function ignores
+		// the audit log if there is no user id. We should find a solution
+		// to make sure this event is tracked.
+		//auditor = api.AGPL.Auditor.Load()
+		//aReq, commitAudit = audit.InitRequest[database.WorkspaceProxy](rw, &audit.RequestParams{
+		//	Audit:   *auditor,
+		//	Log:     api.Logger,
+		//	Request: r,
+		//	Action:  database.AuditActionWrite,
+		//})
```

Comment:
Going to kick fixing this to another PR. 

I commented out rather than deleting it to remind us that we probably should audit log this, but at the moment we cannot.